tests/Localization/LtLtTest.php

```diff
@@ -17,210 +17,415 @@ class LtLtTest extends LocalizationTestCase
 
     const CASES = [
         // Carbon::parse('2018-01-04 00:00:00')->addDays(1)->calendar(Carbon::parse('2018-01-04 00:00:00'))
+        // 'Tomorrow at 12:00 AM'
         'Tomorrow at 00:00',
+
         // Carbon::parse('2018-01-04 00:00:00')->addDays(2)->calendar(Carbon::parse('2018-01-04 00:00:00'))
+        // 'Saturday at 12:00 AM'
         'šeštadienis at 00:00',
```

Comment:
every `at` should be removed.
should be `šeštadienį 00:00`

Reply:
Updated.